data/projects.js

```diff
@@ -250,5 +250,25 @@ coala should support generating metrics for your code.",
 		"tags" : ["Documentation", "Bears", "Testing"],
 		"markdown": "bear_updates.md",
 		"collaborating_projects" : ["coala"]
+	},
+	{
+		"name" : "Convert Bears to Aspects",
+		"desc" : "This project is about annotating results of bears with \
+		\"aspects\" which are like categories allowing to group results.",
+		"requirements" : [
+		"The participant should have one patch to any bear accepted.",
+		"The participant should have understanding of \
+		[cEP-0005](https://github.com/coala/cEPs/blob/master/cEP-0005.md).",
+		"The proposal should contain bears which are planned to be addressed \
+		nd a rough draft on which aspects will be needed and how they will be \
+		implemented."
+		],
+		"difficulty" : "medium",
+		"issues" : [],
+		"mentors" : [],
```

Comment:
let's use sils and userzimmermann for now, @userzimmermann shalt complain if he's not interested but I think he'll be as he's the aspects team leader